pkg/controllers/cspi-controller/controller.go

```diff
@@ -59,6 +59,9 @@ type CStorPoolInstanceController struct {
 	// recorder is an event recorder for recording Event resources to the
 	// Kubernetes API.
 	recorder record.EventRecorder
+
+	// ZcmdExecutor is used to execute ZFS and ZPOOL commands
+	zcmdExecutor zcmd.Executor
```

Comment:
n_p: 
1. `zcmdExecutor` ( Notice that z will start with a small case )
2. I would like the comment to be as -- 
```
// zcmdExecutor is an interface that knows to execute ZFS and ZPOOL commands.
// This is useful in mocking.
```
The comment explains the idea of dependency injection.

Reply:
Updated